cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -54,7 +57,8 @@ func TestEvaluateExecute(t *testing.T) {
 			"--repository", "golang/plain",
 		},
 
-		ExpectedOutputContains: `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=6, coverage-statement=1, files-executed=1, response-no-error=1, response-no-excess=1, response-not-empty=1, response-with-code=1`,
+		ExpectedOutputContains:      `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=6, coverage-statement=1, files-executed=1, response-no-error=1, response-no-excess=1, response-not-empty=1, response-with-code=1`,
```

Comment:
Please redo the logic for checking the output. do a callback for ExpectedOutputValidate and do a Contains for the one we want and then the logic for counting the models. that way we are not overcrowding our fields over time with more and more forms of validation.